### ISSUE TITLE
[hakari] Warning about no edititon set in a newly created workspace hack crate

### ISSUE THREAD

**Issue by VorpalBlade** [guppy-rs/guppy#318]
An edition should probably be set in the default tempalte

```console
$ cargo hakari init crates/paketkoll_workspace_hack
$ cargo hakari generate
$ cargo hakari manage-deps
$ cargo build
warning: /home/arvid/src/paketkoll/crates/paketkoll_workspace_hack/Cargo.toml: no edition set: defaulting to the 2015 edition while the latest is 2021
[...]
```

* rustc 1.81.0 (from rustup)
* Arch Linux
* Using workspace resolver 2 in my virtual workspace.

**Comment by sunshowers** [comment]
Thanks -- yes, agreed. #319 addresses this.

**Comment by sunshowers** [comment]
Fixed in cargo-hakari 0.9.33, which will be out within 15 minutes.
